android/quest/src/main/java/org/smartregister/fhircore/quest/ui/usersetting/UserSettingScreen.kt

```diff
@@ -380,6 +381,16 @@ fun UserSettingScreen(
           modifier = modifier.padding(top = 8.dp).align(Alignment.CenterHorizontally),
         )
 
+        if (implementationGuideVersion != null) {
+          Text(
+            color = contentColor,
+            fontSize = 16.sp,
+            text = stringResource(id = R.string.ig_version, implementationGuideVersion),
+            modifier =
+              modifier.padding(bottom = 12.dp, top = 2.dp).align(Alignment.CenterHorizontally),
+          )
+        }
+
```

Comment:
tl;dr this looks good, but depends on the change to the text I suggested in the strings file

I think this is *somewhat* but not entirely misleading. An IG might be in other parts of the app too, so this is probably going to be the version of an IG, but really it's the app, part of me thinks it would better to have this go in the app ID, if we're using sync by ID -- but on the other hand, for debugging purposes it would be more useful to see both the app version and the IG app wrapper version.

Reply:
Again. I will remove this because we reverted to use app_version to fetch the ImplementationGuide